src/main/java/com/datastax/cdm/cql/statement/TargetUpdateStatement.java

```diff
@@ -42,35 +42,39 @@ protected BoundStatement bind(Row originRow, Row targetRow, Integer ttl, Long wr
             boundStatement = boundStatement.set(currentBindIndex++, writeTime, Long.class);
         }
 
-        Object bindValue = null;
+        Object bindValueOrg = null;
```

Comment:
#nitpick on the variable names used. We use `origin` / `destination` here and `source` / `target` else.